arch/arm64/start.S

```diff
@@ -266,6 +266,24 @@ arm_reset:
     b       .Lmap_range_one_table_loop
 
 .Linitial_mapping_done:
+    ldr     tmp, =MMU_TCR_FLAGS_BASE
```

Comment:
suggestion: put a quick comment here as to what this block is for. Something to the effect of 'compute the base TCR configuration and save away in a global for future use.'

Reply:
Done